### PR TITLE
Create angular2_universal.yml

### PR DESCRIPTION
**Pull request by PatrickJS**

### CODE REVIEW

File: _data/projects/angular2_universal.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Universal (isomorphic) javascript support for Angular 2
+site: https://github.com/angular/universal
+tags:
+- JavaScript
+- web
+- Angular 2
+- Universal JavaScript
+- Isomorphic JavaScript
+upforgrabs:
+  name: hotlist: community-help
```
Comment (shiftkey) [inline _data/projects/angular2_universal.yml]: The `:` here is causing a parsing issue. You'll need to wrap this value in double-quotes.

**Comment by shiftkey** [comment]
Apologies for the delay on reviewing this. I found a parsing issue which will mean the project won't appear correctly on the site if this was merged.

Aside from that this is good to go!

**Comment by PatrickJS** [comment]
@shiftkey done

**Comment by shiftkey** [comment]
Thanks!

**Pull request merged**